contents/handbook/product/product-stages.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Product stages
+sidebar: Handbook
+showTitle: true
+---
+
+We have multiple teams working on multiple products at various stages, it's essential to have a clear understanding of our focus and priorities. This guide helps answer crucial questions related to product development, launches, pricing, and growth. Please note that this is a rough guide, and you should adapt it to your teams and individual products.
+
+It should help address confusion around when to focus on feature parity vs innovative new major features. How much do we need to care about stability vs it being fine to move fast and break things. Should we keep investing in the smaller features or is doing the first 80% fine.
+
+**Caveat:** this guide mainly applies to building products where the category already exists. Defining a new category often requires different ordering/thinking.
+
+## The spectrum of product development
+
+Our products fall under three main categories within the product development spectrum:
+
+1. Minimum Viable Product (MVP)
+2. Product-Market Fit (PMF)
+3. Standalone Best-in-Class
+
+### 1. Minimum Viable Product (MVP)
+
+- Core features are in production; external rollout has started
+- Some rough edges may be present
+- Initially free to gather feedback and validate demand
+
+To reach the MVP stage:
+
+- Validate demand and identify crucial features through early customer conversations
+- Move fast, things might break, but fix issues quickly
+- Ruthlessly cut scope
+- Work with a small team (about 2 people)
+- Should move quickly based on gut/existing solutions in the market.
+
+When to go from an idea to an MVP:
+
+- TODO: add a link to the new section on which products we'll want to launch
+- Someone has conviction in the idea and envisions a rough path to PMF for our ICP
+- Take bets on ideas, understanding that some products may not reach PMF and could be deprecated
+
+### 2. Product-Market Fit (PMF)
+
+- Strong traction with Ideal Customer Profile (ICP) customers - we are commonly referenced in this category
+- Near feature parity with established tools
+- Differentiation mainly through integration with our other tools (If we can’t get to PMF through this then we consider doing bigger feature innovations)
+- Scalable, reliable, and profitable
+- API access for everything
```

Comment:
Because of Django Rest Framework we often get this for free. However, our API experience is often still lacking, so it's not clear to me how important this is.